source/includes/_cluster.md

```diff
@@ -0,0 +1,419 @@
+# Cluster
+
+## The ClusterState Resource
+
+<aside class="notice">
+This section describes API features that are part of the Cloudify premium edition
```

Comment:
This section describes **the** API features that are ~part of~ included with the Cloudify ~premium edition~ **Premium Edition.**